easybuild/easyconfigs/a/Automake/Automake-1.16.5.eb

```
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'Automake'
+version = '1.16.5'
+
+homepage = 'https://www.gnu.org/software/automake/automake.html'
+
+description = "Automake: GNU Standards-compliant Makefile generator"
+
+toolchain = SYSTEM
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['07bd24ad08a64bc17250ce09ec56e921d6343903943e99ccf63bbf0705e34605']
+
+dependencies = [
+    ('Autoconf', '2.71'),
+]
+
+osdependencies = [
+    'perl-Thread-Queue',
```

@dithwick in ubuntu, `Thread::Queue` is already available by default, and there is no package that provides it.
so, this will fail in ubuntu (I suspect debian too?).

a workaround could be to add a package that is always (automatically) installed in ubuntu/debian, but does not exist in RPM distros. For example, `debianutils` is available in ubuntu, but not in Fedora36 and CentOS 7. should ideally also check other distros and versions.

```suggestion
    # Thread::Queue is already available by default in Ubuntu
    # providing a package that is automatically installed as workaround
    ('perl-Thread-Queue', 'debianutils'),
```

the proper way to deal with this would be to add support for specifying the distros, but that's not (yet) implemented

---

the alternative would be to just comment out the osdeps.

---

Hi @smoors,

I think the first option is probably the best (i.e. the dependency on debianutils).